NEWS.md

```diff
@@ -1,3 +1,68 @@
+# New in snapd 2.64:
+* Support building snapd using base Core22 (Snapcraft 8.x)
+* FIPS: support building FIPS complaint snapd variant that switches to FIPS mode when the system boots with FIPS enabled
+* AppArmor: update to AppArmor 4.0.1
+* AppArmor: support AppArmor snippet priorities
+* AppArmor prompting: add checks for prompting support, include prompting status in system key, and restart snapd if prompting flag changes
+* AppArmor prompting: include prompt prefix in AppArmor rules if prompting is supported and enabled
+* AppArmor prompting: add common types, constraints, and mappings from AppArmor permissions to abstract permissions
+* AppArmor prompting: add path pattern parsing and matching
+* Registry views (experimental): rename from aspects to registries
+* Registry views (experimental): support reading registry views using snapctl
+* Registry views (experimental): restrict view paths from using a number as first character and view names to storage path style patterns
+* Snap components: support installing snaps and components from files at the same time (no REST API/CLI)
+* Snap components: support downloading components related assertions from the store
+* Snap components: support installing components from the store (no REST API/CLI)
+* Snap components: support removing components (REST API, no CLI)
+* Snap components: started support for component hooks
+* Snap components: support kernel modules as components
+* Refresh app awareness (experimental): add data field for /v2/changes REST API to allow associating each task with affected snaps
+* Refresh app awareness (experimental): use the app name from .desktop file in notifications
+* Allow re-exec when SNAP_REEXEC is set for unlisted distros to simplify testing
+* Generate GNU build ID for Go binaries
+* Add missing etelpmoc.sh for shell completion
+* Do not attempt to run snapd on classic when re-exec is disabled
+* Packaging/build maintenance for Debian sid, Fedora, Arch, openSuse
+* Add snap debug api command to enable running raw queries
+* Enable snap-confine snap mount directory detection
+* Replace global seccomp filter with deny rules in standard seccomp template
+* Remove support for Ubuntu Core Launcher (superseded by snap-confine)
+* Support creating pending serial bound users after serial assertion becomes available
+* Support disabling cloud-init using kernel command-line
+* In hybrid systems, apps can refresh without waiting for restarts required by essential snaps
+* Ship snap-debug-info.sh script used for system diagnostics
+* Refresh app awareness (experimental): give snap-refresh-observe interface access to /v2/snaps/{name} REST API
```

Comment:
Tiny nitpick, but this could be moved up to line 21 to be grouped with similar entries

```suggestion
* Refresh app awareness (experimental): give snap-refresh-observe interface access to /v2/snaps/{name} endpoint
```

Reply:
thanks! fixed and added to group